src/Blockcore.Indexer/appsettings.json

```diff
@@ -41,6 +41,6 @@
       "AverageInterval": 10,
 
       // Store the trx hex in mongo storage or read it from RPC
-      "StoreRawTransactions": true
+      "StoreRawTransactions": false
```

Comment:
This we have to remember to flip on production deployment, @sondreb FYI David proposes we will not store trx data in store by default but read it form the client, IMO for prod deployments that the indexer will use we should store them in mongo

Reply:
What does it mean "by default"? You mean during development and running locally on developer computer?

This setting is set by the published chain info JSON file, so we'll just make sure all JSON files there has this set to true.